src/all/foamgirl/src/eu/kanade/tachiyomi/extension/all/foamgirl/FoamGirl.kt

```diff
@@ -119,7 +147,11 @@ class FoamGirl() : ParsedHttpSource() {
         }
     }
 
+    private fun String.isNumber() = NUMB_REGEX.matches(this)
```

Comment:
Avoid regex if possible.

This is equivalent to `^\d+$`:
```suggestion
    private fun String.isNumber() = isNotEmpty() && all { it.isDigit() }
```